<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>get_multicast</class>
 <widget class="QMainWindow" name="get_multicast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>get_multicast</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="scan_ip">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>60</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>扫描</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>数据筛选设置</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>216</width>
       <height>108</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>数据筛选1：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="n_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{qproperty-alignment:AlignHCenter;}
edit-&gt;setAlignment( Qt::AlignHCenter);</string>
          </property>
          <property name="text">
           <string>n</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="x_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{qproperty-alignment:AlignHCenter;}
edit-&gt;setAlignment( Qt::AlignHCenter);</string>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="data_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{qproperty-alignment:AlignHCenter;}
edit-&gt;setAlignment( Qt::AlignHCenter);</string>
          </property>
          <property name="text">
           <string>data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>数据筛选2：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="n_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{qproperty-alignment:AlignHCenter;}
edit-&gt;setAlignment( Qt::AlignHCenter);</string>
          </property>
          <property name="text">
           <string>n</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="x_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{qproperty-alignment:AlignHCenter;}
edit-&gt;setAlignment( Qt::AlignHCenter);</string>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="data_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{qproperty-alignment:AlignHCenter;}
edit-&gt;setAlignment( Qt::AlignHCenter);</string>
          </property>
          <property name="text">
           <string>data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>25</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>data在数据位的第n位，连续x位</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>端口设置</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>191</width>
       <height>84</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>组播端口:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="port">
          <property name="styleSheet">
           <string notr="true">QLineEdit{qproperty-alignment:AlignHCenter;}
edit-&gt;setAlignment( Qt::AlignHCenter);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>组播  IP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="IP">
          <property name="styleSheet">
           <string notr="true">QLineEdit{qproperty-alignment:AlignHCenter;}
edit-&gt;setAlignment( Qt::AlignHCenter);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
